jwst/pipeline/calwebb_image3.py

```diff
@@ -32,6 +32,7 @@ class Image3Pipeline(Pipeline):
     class_alias = "calwebb_image3"
 
     spec = """
+    on_disk = boolean(default=False)  # Preserve memory using temporary files
```

Comment:
change to `in_memory` for compatibility with individual steps?

Reply:
I think consistency would be good here but I don't have a strong opinion one way (`in_memory`) or the other (`on_disk`). If there are pars files that have `in_memory` (I haven't found any) that might make sticking with `in_memory` the easier option.